js/scenes/DevRoutes/DevRoutesContainer.js

```diff
@@ -12,11 +14,23 @@ class DevRoutesContainer extends Component {
     }
   };
 
+  _logOut = () => {
+    this.props.logOut();
+  };
+
   render() {
-    return <DevRoutes />;
+    return <DevRoutes logOut={this._logOut} />;
   }
 }
 
-// DevRoutesContainer.propTypes = {};
+const mapDispatchToProps = dispatch => ({
+  logOut: () => {
+    dispatch(logOut());
+  }
+});
+
+DevRoutesContainer.propTypes = {
+  logOut: PropTypes.func.isRequired
+};
 
-export default DevRoutesContainer;
+export default connect(null, mapDispatchToProps)(DevRoutesContainer);
```

Comment:
Nicely done, Scott! 
You do not need to pass null if you are not doing mapStateToProps, you can just pass mapDispatchToProps and it will still work.